errors.go

```diff
@@ -69,6 +69,39 @@ func (e *errRequiredFlags) getMissingFlags() []string {
 	return e.missingFlags
 }
 
+type mutuallyExclusiveGroup struct {
+	flag1Name string
+	flag2Name string
+}
+
+func (e *mutuallyExclusiveGroup) Error() string {
+	return fmt.Sprintf("option %s cannot be set along with option %s", e.flag1Name, e.flag2Name)
+}
+
+type mutuallyExclusiveGroupRequiredFlag struct {
+	flags *FlagExGroup
+}
+
+func (e *mutuallyExclusiveGroupRequiredFlag) Error() string {
+
+	var missingFlags []string
+	for _, grpf := range e.flags.Flags {
+		var grpString []string
+		for _, f := range grpf {
+			grpString = append(grpString, f.Names()...)
+		}
+		if len(e.flags.Flags) == 1 {
+			err := errRequiredFlags{
+				missingFlags: grpString,
+			}
+			return err.Error()
+		}
+		missingFlags = append(missingFlags, strings.Join(grpString, " "))
+	}
+
+	return fmt.Sprintf("one of these flags needs to be provided %s", strings.Join(missingFlags, "\n"))
```

Comment:
so that the error text doesn't use multiple lines by default

```suggestion
	return fmt.Sprintf("one of these flags needs to be provided: %s", strings.Join(missingFlags, ", "))
```